client/web/src/Structure/Admin/State/action.js

```diff
@@ -230,25 +231,173 @@ const updateCrudTopics = (payload, id) => async (dispatch) => {
 
 }
 
+const getQuestionsLoading = () => ({
+    type: adminConstants.GET_ALL_QUESTIONS_LOADING,
+  });
+  
+  const getQuestionsSuccess = (payload) => ({
+    type: adminConstants.GET_ALL_QUESTIONS_SUCCESS,
+    payload,
+  });
+  
+  const getQuestionsFailure = (data) => ({
+    type: adminConstants.GET_ALL_QUESTIONS_FAILURE,
+    payload: data
+  });
+  
+  const getQuestionsRequest = (page = 1, limit = 10) => (dispatch) => {
+    dispatch(getQuestionsLoading());
+    const token = (localStorage.getItem('token'))
+    let url = `${QUESTION_URL}/all/?page=${page+1}&limit=${limit}`
+
```

Comment:
default value of page is one and you are increasing it by 1 so you will get second page. so check this